docs/plugins/plugin-script.asciidoc

```diff
@@ -127,6 +127,24 @@ return the following exit codes:
 `70`:: any other error
 
 [float]
+=== Batch mode
+
+Certain plugins require more privileges than those provided by default in core
+Elasticsearch. These plugins will list the required privileges and ask the
+user for confirmation before continuing with installation.
```

Comment:
Is it worth being explicit that if you run in batch mode you automatically grant all requested permissions?